.github/workflows/windows-release.yml

```
@@ -0,0 +1,377 @@
+          
+          if ($env:UPLOAD_LATEST -eq "true") {
+              aws s3api put-object-acl --bucket $bucket --key $latestKey --acl public-read --region $region
+          }
+          
+          Write-Host "✅ Set public read permissions for auto-updater access"
+          
+          # Store installer path for GitHub release
+          "SIGNED_INSTALLER_PATH=$($signedInstaller.FullName)" | Out-File -FilePath $env:GITHUB_ENV -Append
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)